### ISSUE TITLE
Issue with spaces in file pathname?

### ISSUE THREAD

**Issue by GH4Arlene** [dangmai/prettier-plugin-apex#1702]
### Self-Troubleshooting Checklist

- [X] Are you running Prettier Apex >= 2.0.0? If Yes, have you gone through the [Upgrade Guide](https://github.com/dangmai/prettier-plugin-apex/wiki/Upgrading-to-Prettier-Apex-v2)?
- [X] Are you running Java >= 11? If No, you need to install a newer Java version.
- [X] Are you running VSCode and recently upgraded Prettier or Prettier Apex? If Yes, have you restarted VSCode after the upgrade?

### Describe your issue

I am unable to format Apex files with prettier.  All other languages work fine.  The issue appears to be that my user directory contains a space.  The error message is 

'c:\Users\Arlene' is not recognized as an internal or external command, operable program or batch file.    

The stack trace is:

```
    at ChildProcess.<anonymous> (file:///c:/Users/Arlene%20Smith/VSCodeProjects/devt/node_modules/prettier-plugin-apex/dist/src/parser.js:38:24)
    at ChildProcess.emit (node:events:519:28)
    at maybeClose (node:internal/child_process:1105:16)
    at ChildProcess._handle.onexit (node:internal/child_process:305:5)
```

Is there a workaround for this?

### System Info

```shell
System:
    OS: Windows 11 10.0.26100
    CPU: (20) x64 13th Gen Intel(R) Core(TM) i7-13700H
    Memory: 44.50 GB / 63.72 GB
  Binaries:
    Node: 20.14.0 - C:\Program Files\nodejs\node.EXE
    npm: 10.7.0 - C:\Program Files\nodejs\npm.CMD
  Languages:
    Java: 17.0.11
  npmPackages:
    @prettier-apex/apex-ast-serializer-win32-x64:  2.2.0
    @prettier/plugin-xml: ^2.2.0 => 2.2.0
    prettier: ^3.3.3 => 3.3.3
    prettier-plugin-apex: ^2.2.0 => 2.2.0
```

**Comment by dangmai** [comment]
Hi there, thank you for reporting this issue. I have fixed this and released a new version `v2.2.1`. Could you try that and let me know if it fixes the problem for you?

**Comment by GH4Arlene** [comment]
Yes, it is fixed.  Thank you so much for the quick turn-around!
